portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -139,13 +139,13 @@
 	<td></td>
 </tr>
 <tr>
-	<td>FILTER_OPTION</td>
-	<td>//div[@class='dropdown-menu show']//*[@class='dropdown-item' and contains(.,'${key_filter}')]</td>
+	<td>SELECT_TYPE_PROVIDER</td>
+	<td>//button[@class="dropdown-item"]//span[contains(.,'${key_type}')]</td>
 	<td></td>
 </tr>
 <tr>
-	<td>SELECT_TYPE_PROVIDER</td>
-	<td>//button[@class="dropdown-item"]//span[contains(.,'${key_type}')]</td>
+	<td>FILTER_OPTION</td>
+	<td>//div[@class='dropdown-menu show']//*[@class='dropdown-item' and contains(.,'${key_filter}')]</td>
 	<td></td>
```

Comment:
Hey @andreluis2022 why were these changes made? I don't see them being used in your test 🤔

Reply:
Done!!